yarn.lock

```diff
@@ -15832,10 +15869,10 @@ styled-jsx@3.3.2, styled-jsx@<3.3.3:
     stylis "3.5.4"
     stylis-rule-sheet "0.0.10"
 
-styled-jsx@^3.2.2, styled-jsx@^3.4.1:
-  version "3.4.1"
-  resolved "https://registry.yarnpkg.com/styled-jsx/-/styled-jsx-3.4.1.tgz#5b12c5e841cec3d7310f5ab6b94c3c8df665caf5"
-  integrity sha512-Jl1pcQVTnHdsfePHxZ0GgxxgU3+KgiDKj9QTp2vZ0eifg2cgNV5RFdgfxcXvJyspsynUhF32opNZ4jmM1EIROA==
+styled-jsx@^3.2.2, styled-jsx@^3.4.4:
+  version "3.4.4"
```

Comment:
Is this really the version you want?
In Analytics we're resolving this to a lower version, since there's a bug in the newer one. https://github.com/dhis2/analytics/blob/ba3d39af36117916ea39459adb0027652dd3c0ed/package.json#L75